### PR TITLE
Updated Readme for scheduler functionality

### PR DESCRIPTION
**Pull request by Rits1272**

### CODE REVIEW

**Comment by Rits1272** [comment]
Closes #226. Please review @rpotter12

File: README.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ python3 -m wplay -wtb "AnyName"
 ```shell
 python3 -m wplay -pull "AnyName"
 ```
+8. Schedule - Used to schedule a message to deliver it at a specified time
+```
```
Comment (rpotter12) [inline README.md]: Codacy Issue found: [[fenced-code-flag] Missing code-language flag](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5214556)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5214556&bid=17098567)

File: README.md [hunk 2]
```diff
@@ -73,6 +73,10 @@ python3 -m wplay -wtb "AnyName"
 ```shell
 python3 -m wplay -pull "AnyName"
 ```
+8. Schedule - Used to schedule a message to deliver it at a specified time
```
Comment (rpotter12) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5214556)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5214556&bid=17098567)

File: README.md [hunk 3]
```diff
@@ -73,6 +73,11 @@ python3 -m wplay -wtb "AnyName"
 ```shell
 python3 -m wplay -pull "AnyName"
 ```
+8. Schedule - Used to deliver a message at a scheduled time.
```
Comment (rpotter12) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5214556)

**Pull request merged**